pkg/kubelet/dockershim/network/cni/cni.go

```diff
@@ -311,10 +314,21 @@ func (plugin *cniNetworkPlugin) addToNetwork(network *cniNetwork, podName string
 		return nil, err
 	}
 
+	if res == nil {
+		glog.Warningf("CNI plugin gives empty network result")
+		return nil, nil
+	}
+
+	curRes, err := cnicurrent.NewResultFromResult(res)
```

Comment:
Should also log result conversion errors here?  Otherwise the pod IP cache will be empty for the pod even though the CNI call succeeded.